### PR TITLE
Add readme deprecation info for track1

### PR DESCRIPTION
**Pull request by KieranBrantnerMagee**
Add a snippet to the head of the readme indicating the pending deprecation, as well as pointers to get ahead of that process.

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## 0.50.3 (unreleased)
 
+> **NOTE**: Starting with the GA release of version 7.0.0, (Currently in preview) this package will be deprecated.
```
Comment (yunhaoling) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:5]: nitpick: should `(currently in preview)` be placed in front of the comma?

```suggestion
> **NOTE**: Starting with the GA release of version 7.0.0 (currently in preview), this package will be deprecated.
```

File: sdk/servicebus/azure-servicebus/README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Microsoft Azure Service Bus SDK for Python
 
+> **NOTE**: Starting with the GA release of version 7.0.0, (Currently in preview) this package will be deprecated.
```
Comment (yunhaoling) [inline sdk/servicebus/azure-servicebus/README.md:3]:
```suggestion
> **NOTE**: Starting with the GA release of version 7.0.0 (currently in preview), this package will be deprecated.
```

File: sdk/servicebus/azure-servicebus/CHANGELOG.md (hunk 1 above)
Comment (annatisch) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md:5]: In the Cosmos notes we changed "GA" -> "stable", as we weren't whether "GA" was too much of a corporate acronym...but up to you on this one.

Comment (KieranBrantnerMagee) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md:5]: I think this (your suggestion) is technically more grammatically correct.  Nice, swapping.

**Pull request merged**